docs/user-guide/third-party-addons/opentracing.md

```diff
@@ -93,6 +93,9 @@ jaeger-tracer-baggage-header-prefix
 # specifies the port to use when uploading traces, Default 8126
 datadog-collector-port
 
+# specifies the url to use for submitting traces. If set, this will be used instead of datadog-collector-host / datadog-collector-port.
```

Comment:
Please add some example also, or at least the comment "Must be a valid URL." the same way jaeger have :)

Reply:
Added additional examples and explanation.